src/vs/workbench/parts/preferences/browser/media/settingsEditor2.css

```diff
@@ -491,19 +491,6 @@
 	padding-top: 7px;
 }
 
-.settings-editor > .settings-body .settings-feedback-button {
-	color: rgb(255, 255, 255);
```

Comment:
is this supposed to be a part of the PR?

Reply:
@sbatten it slipped in